pymc/variational/opvi.py

```diff
@@ -943,9 +960,9 @@ def _new_initial(self, size, deterministic, more_replacements=None):
             if deterministic:
                 return at.ones(shape, dtype) * dist_map
             else:
-                return getattr(self._rng, dist_name)(size=shape)
+                return getattr(self.srng, dist_name)(size=shape)
```

Comment:
Same applies here, if you can just do `getattr(at.random, "normal")`, you don't have the issues with the eager `default_update`